docs/requirements-docs.txt

```diff
@@ -1,6 +1,6 @@
 # See tox.ini for documentation on this file.
 funcsigs ; python_version < '3'
 six
-Sphinx
+Sphinx<2
```

Comment:
Was Sphinx 2.0 causing an issue?  I don't see this in the master branch.

Reply:
Yes, but only on the 2.x branch.